examples/end-to-end-grpc-gateway/README.md

```diff
@@ -97,7 +97,7 @@ Create a book for the shelve with the id `1`:
       http://localhost:8081/shelves/1/books \
       -H 'Content-Type: application/json' \
       -d '{
-      "application_json": {
+      "book": {
```

Comment:
I believe that "book" in this case should be implicit and that the "author", "name", and "title" values should be at the top level. As a check on this, I used swagger-codegen to generate a nodejs server for our bookstore.yaml and found the code below in the generated server. Notice the structure of the example:
```
/**
 * Create a new book on the shelf.
 *
 * body Book Book to create.
 * shelf Long ID of the shelf where the book should be created.
 * returns book
 **/
exports.createBook = function(body,shelf) {
  return new Promise(function(resolve, reject) {
    var examples = {};
    examples['application/json'] = {
  "author" : "author",
  "name" : "name",
  "title" : "title"
};
    if (Object.keys(examples).length > 0) {
      resolve(examples[Object.keys(examples)[0]]);
    } else {
      resolve();
    }
  });
}
```